include/template.php

```diff
@@ -263,7 +263,9 @@ function parseTags($line, $vars) {
 //
 // Renders the templates for the given view
 
-function renderTemplate($view) {
+function renderTemplate($view,$readmepath=null) 
```

Comment:
Spaces and camel-case make things read better, like in your other function: `$view, $readmePath = null`

Reply:
Done.

Reply:
Please add the spaces as well, like in your other functions:

```
function renderTemplate($view, $readmePath = null)
```

```
if ($readmePath == null)
```

Reply:
Done.